### PR TITLE
Bounties framework

### PR DESCRIPTION
**Pull request by Kmc2000**
This PR adds in consoles for bounty management. Anyone can make one or claim one if they have the money (theyre not paid out automatically, and must be confirmed manually)

Clock said they want it as a program, but here's a first draft to see if the concept's alright.

DEMOS:

https://cdn.discordapp.com/attachments/265411250341543936/556935651782688809/bounties.mp4

Relevant issue: https://github.com/discordia-space/CEV-Eris/issues/3125

### CODE REVIEW

File: code/controllers/subsystems/bounties.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SUBSYSTEM_DEF(bounty)
+	name = "Bounty"
+	flags = SS_NO_FIRE //It's just a storage / manager object
```
Comment (nichlas0010) [inline code/controllers/subsystems/bounties.dm]: so why is it not a GLOB instead of a subsystem?

Comment (Kmc2000) [reply code/controllers/subsystems/bounties.dm]: Subsystems make it look cool and impressive 😎

It didn't occur

Comment (ACCount12) [reply code/controllers/subsystems/bounties.dm]: I don't really like the idea of everything being a subsystem. And this? It really doesn't need to be one.

**Comment by ACCount12** [comment]
Once the subsystem is removed, I can merge it as is, but only as a temporary solution.

**Comment by Kmc2000** [comment]
You'll need to map these in somehow. I'm told you have to """reserve the map""" because no mapmerge.

**Comment by Kmc2000** [comment]
This is ready as a first version. I need someone who can nanoUI to help make it a program though.

**Pull request merged**